tincr/io/design/rapidsmith.tcl

```diff
@@ -0,0 +1,238 @@
+        puts "0\n$expected_inpin\n$actual_inpin\n$expected_outpin\n$actual_outpin\n"
+    }
+    
+    puts 1
+}
+
+## Test that the specified Bel is a static source bel (i.e. the configuration equation is 0 or 1).
+#
+# @param bel Name of a bel
+# @return 1 if the bel is a static source, 0 otherwise
+proc ::tincr::test_static_sources { bel } {
+    
+    set config [get_property CONFIG.EQN [get_bels $bel]]
+    
+    if {[regexp {(O[5,6])=[0,1] ?} $config -> pin] } {
+        puts 1
+    }
+    puts 0
+}
+
+## Prints the values of the properties in {@code property_list} for the specified cell. 
+#   The properties are printed in the following format: <br>
+#    "propertyName1 propertyValue1\n" <br>
+#    "propertyName2 propertyValue2\n" <br>
+#     ...
+#
+#   This function is used to verify cell properties in RapidSmith
+# @param cell Name of a cell
+# @param property_list list of property names
+proc ::tincr::get_property_values {cell property_list} {
+    
+    set cell [get_cells $cell]
+    
+    set return_string ""
+    foreach property $property_list {
+        set return_string "$return_string$property [get_property $property $cell]\n"
+    }
+    
+    puts -nonewline $return_string
+}
+
+## Prints the placement information of a cell to standard out in the following format: <br>
+#   (1) sitename/belname ; what bel the cell is placed on <br>
+#   (2) cellpin1 belpin1 belpin2 ; the cell pin to bel pin mappings for a cell pin <br>
+#   (3) cellpin2 belpin1 belpin2 ; <br> 
+#   ... <br>
+#
+#   This function is used to verify that a cell has been imported into RapidSmith correctly.Used for testing that cell of a design have been imported correctly
+#
+# @param cell Name of the cell to  
+proc ::tincr::get_cell_placement_info {cell} {
+   
+    set cell [get_cells $cell]
+    
+    # Get what bel the cell is placed on
+    if {[get_property STATUS $cell] != "PLACED"} {
+        set returnString "\n"
+    } else {
+        set site [get_sites -of $cell]
+        set belPlacement [lindex [split [get_bels -of $cell -quiet] "/"] end]
+    
+        if { [get_property IS_PAD $site] } {
+            set site [get_property NAME [get_package_pins -quiet -of_object $site]]
+        }
+        set returnString "$site/$belPlacement\n"
+    }
+    
+    # populate the pin mappings for each cell pin of the cell
+    foreach pin [get_pins -of [get_cells $cell]] {
+        set returnString "$returnString[get_property REF_PIN_NAME $pin] [get_bel_pins -of $pin -quiet]\n"
+    }
+    
+    puts -nonewline $returnString
+}
+
+## Prints the wires, cell pins, and bel pins of the net to standard out in the following format: <br>
+#
+#   "wire1 wire2 ... \n" <br>
+#   "pin1 pin2 ... pin3" <br>
+#   "belpin1 belpin2 ... belpin3" <br>
+#
+#   This function is used to verify that a net has been imported correctly into RapidSmith
+#
+proc ::tincr::get_physical_net_info {net} {
+    set_param tcl.collectionResultDisplayLimit 100000
```

Comment:
I added two functions in util.tcl, `set_tcl_display_limit `and `reset_tcl_display_limit` and modified my unit tests to call these functions to set and reset the display limit.